client-src/utils/log.js

```diff
@@ -18,7 +18,8 @@ setLogLevel(defaultLevel);
 
 const log = logger.getLogger(name);
 
-const logEnabledFeatures = (features) => {
+const logEnabledFeatures = (features, logLevel = "info") => {
+  setLogLevel(logLevel);
```

Comment:
Let's avoid it and just move `logEnabledFeatures` here https://github.com/webpack/webpack-dev-server/blob/9baea53c469b3a4486ab1991c52a12077ad01cd9/client-src/index.js#L113

Reply:
Done!